docs/docs/admin/native-install.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Installing Anubis with a native package
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Install the Anubis package using your package manager of choice:
+
+<Tabs>
+  <TabItem value="deb" label="Debian-based (apt)" default>
+  
+Install Anubis with `dpkg -i`:
+
+```text
+dpkg -i ./anubis-$VERSION-$ARCH.deb
```

Comment:
```suggestion
apt install ./anubis-$VERSION-$ARCH.deb
```
This automatically pulls all dependencies. Otherwise you sometimes need to run apt install --fix-broken which is unnecessary.

Reply:
TIL they added that. Neat!